### DISCUSSION TITLE
Welcome to RobimGH V1.1.0 Beta Release 🌱

### DISCUSSION THREAD

**Discussion by ziyueg** [RoboticPlus/Community-Hub#3] [Announcements]
## Welcome to RobimGH V1.1.0 Beta Release 🌱
We are pleased to share with you the beta release of RobimGH V1.1.0. This update brings a suite of new features and improvements designed to refine the user experience and enhance performance. While we're excited about these developments, we recognize this is a beta version, and your feedback will be invaluable for refining these features. Here's an overview of what's new:

### 1. Asynchronous Robot Loader 🔄
To enhance your Grasshopper experience, we've introduced an asynchronous robot loader. This change aims to prevent the Grasshopper interface from freezing, with a status message to keep you informed about the loading process. We hope this makes your workflow smoother, allowing for uninterrupted creativity.

### 2. Customizable External Values 🛠️
Flexibility in planning is now improved with the ability to customize external values. This feature lets you lock in external values during planning, combining the best of both freeze planning and auto planning. It's a step towards more controlled and precise simulation settings.

### 3. Joint Targets Addition 🎯
This update includes the ability to specify joint targets, offering finer control over the robot's movement across desired axis values, including externals. The integration of joint and cartesian targets for planning introduces a new level of flexibility in defining your robot's path.

### 4. Optimized Simulation Performance 🏎️
We have worked on optimizing simulation performance to make Robim Simulation smoother, aiming for a more responsive and enjoyable user experience. This beta version promises significant speed improvements, though we're keen to hear your feedback on performance.

### 5. Detailed Bilingual Error Messages 📄
To aid in understanding and resolving planning errors, we've added more detailed bilingual error messages. This improvement is designed to help users quickly identify and address issues, making the learning curve a little less steep.

### 6. Bug Fixes and Performance Improvements 🛠️
This release also includes various bug fixes and performance enhancements aimed at providing a more stable and efficient environment. Your feedback on these changes is especially welcome as we strive to polish the overall experience.

## Looking Ahead
As a beta release, V1.1.0 represents our ongoing efforts to develop a robust and user-friendly Grasshopper plugin for robotic planning. We eagerly await your feedback and insights, which are crucial for the continued improvement of RobimGH. Together, we can shape a tool that meets the community's needs and exceeds expectations.

**Thank you for joining us on this journey, and we look forward to your contributions and feedback.**

_RoboticPlus.AI Team_

**Comment by grasscaograss** [comment]
AWESOME 😊
